cache/test_cache.py

```diff
@@ -810,6 +810,126 @@ def test_cache_POST_request_not_cached(self, name, extra_header):
         self.assertNotIn("age", client.last_response.headers.keys())
         self.assertEqual(len(server.requests), 2)
 
+    @parameterize.expand(
+        [
+            param(
+                name="GET_POST",
+                first_method="GET",
+                second_method="POST",
+                second_headers=[],
+                sleep_interval=0,
+            ),
+            param(
+                name="GET_GET",
+                first_method="GET",
+                second_method="GET",
+                second_headers=[("cache-control", "max-age=1")],
+                sleep_interval=2,
+            ),
+            param(
+                name="GET_HEAD",
+                first_method="GET",
+                second_method="HEAD",
+                second_headers=[("cache-control", "max-age=1")],
+                sleep_interval=2,
+            ),
+            param(
+                name="POST_POST",
+                first_method="POST",
+                second_method="POST",
+                second_headers=[],
+                sleep_interval=0,
+            ),
+            param(
+                name="POST_GET",
+                first_method="POST",
+                second_method="GET",
+                second_headers=[("cache-control", "max-age=1")],
+                sleep_interval=2,
+            ),
+            param(
+                name="POST_HEAD",
+                first_method="POST",
+                second_method="HEAD",
+                second_headers=[("cache-control", "max-age=1")],
+                sleep_interval=2,
+            ),
+        ]
+    )
+    def test_several_entries(
+        self, name, first_method, second_method, second_headers, sleep_interval
+    ):
```

Comment:
Please add docstring for these cases. RFC point etc. Probably, we can merge these tests with [these tests](https://github.com/tempesta-tech/tempesta-test/blob/bd7582c353afb8e724844ab337501f00f8e28c67/cache/test_cache.py#L652).

Reply:
Add comments. This tests check how Tempesta FW satisfy request from cache, if there are several cache entries